core/templates/pages/about-page/about-page.component.ts

```diff
@@ -71,6 +71,37 @@ export class AboutPageComponent implements OnInit, OnDestroy {
     },
   ];
 
+  partnersData = [
```

Comment:
Should we have it a common location, since this data may need to be changed on a more frequent basis?